### PR TITLE
Use sliver-based scrolling in more places

### PR DESCRIPTION
**Pull request by abarth**
This patch uses sliver-based two more gallery demos, the stocks example,
in the date picker, and in markdown.

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie

File: packages/flutter/lib/src/widgets/scroll_view.dart [hunk 1]
```diff
@@ -190,7 +209,7 @@ class ListView extends BoxScrollView {
 
   final double itemExtent;
 
-  final SliverChildListDelegate childrenDelegate;
+  final SliverChildDelegate childrenDelegate;
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/scroll_view.dart]: can you fix the other two cases of this bug in this file while you're at it?

**Comment by Hixie** [comment]
LGTM

File: packages/flutter/lib/src/widgets/scroll_view.dart (hunk 1 above)
Comment (abarth) [inline packages/flutter/lib/src/widgets/scroll_view.dart]: Done.

**Pull request merged**